metrics/internal/cache/pv.go

```diff
@@ -171,6 +227,12 @@ func (p *PersistentVolumeStore) add(pv *corev1.PersistentVolume) error {
 		return fmt.Errorf("failed to get node name for pod: %v", err)
 	}
 
+	childrenCount, err := p.runCephRBDChildrenCountFn(&p.monitorConfig, pv.Spec.CSI.VolumeAttributes["pool"], pv.Spec.CSI.VolumeAttributes["radosNamespace"], pv.Spec.CSI.VolumeAttributes["imageName"])
+	if err != nil {
+		return fmt.Errorf("failed to get children count for image %s/%s: %v", pv.Spec.CSI.VolumeAttributes["imageName"], pv.Spec.CSI.VolumeAttributes["radosNamespace"], err)
```

Comment:
You have the order reversed in your error (as an image belongs to a RNS not the other way around)
Also preferably you should add the pool name as without it there is no way to know which pool is in question
```suggestion
		return fmt.Errorf("failed to get children count for image %s/%s: %v", pv.Spec.CSI.VolumeAttributes["radosNamespace"], pv.Spec.CSI.VolumeAttributes["imageName"], err)
```